### ISSUE TITLE
Gps103 ignoring "help me" Message without location

### ISSUE THREAD

**Issue by ivanfmartinez** [tananaev/traccar#1558]
Hi,
testing an obd tracker inside my home, I have tried to use the sos button which send an "help me" message.
But the message is ignored by the gps103 decoder.

Here is one sample message (IMEI changed)

<pre>imei:359710099999999,help me,,,L,,,113b,,558f,,,,,0,0,,,;</pre>

I have tried to add this message in the Gps103ProtocolDecoderTest, and the error is "position is null"
Apparently the GPS signal is off where I'm testing, the tracker are sending the message and the decoder just ignoring because does not have a valid position.

Is possible to accept this kind of message and just use the last valid location for device and set an attribute indicating this ? Maybe setting "Valid" to false/no.

I have tried with current version and it does not recognize this specific message.
The current version recognize the message that is sent  when the GPS have a valid location, not when the help message come without valid lat/lon.

**Comment by tananaev** [comment]
Those kind of messages will be supported starting from next release. The support is already implemented, so if you don't want to wait, you can compile a build from source code.

**Comment by ivanfmartinez** [comment]
I have tried with the current github code, but does not work.

The simple alive messages update the status time (but are not passed thru the handlers).

But the messages like the one I posted before does not update status time, neither go thru the handlers.
Its important to register when someone pressed the help button on device, and also make the message pass thru the handlers to take care.

In my testing I'm trying to bridge the tracking information to an mqtt server. This will provide many possibilities for integration.

Here part of the log, without decoding.


2015-11-27 10:27:16 DEBUG: [43ADFDC8: 5001 < 189.40.72.181] HEX: 23232c696d65693a3335393731303034393037353039372c413b
2015-11-27 10:27:16 DEBUG: [43ADFDC8: 5001 > 189.40.72.181] HEX: 4c4f4144
2015-11-27 10:27:25 DEBUG: [43ADFDC8: 5001 < 189.40.72.181] HEX: 696d65693a3335393731303034393037353039372c68656c70206d652c2c2c4c2c2c2c313133622c2c353538662c2c2c2c2c302c302c2c2c3b
2015-11-27 10:27:37 DEBUG: [43ADFDC8: 5001 < 189.40.72.181] HEX: 696d65693a3335393731303034393037353039372c68656c70206d652c2c2c4c2c2c2c313133622c2c353538662c2c2c2c2c302c302c2c2c3b
2015-11-27 10:27:42 DEBUG: [43ADFDC8: 5001 < 189.40.72.181] HEX: 696d65693a3335393731303034393037353039372c747261636b65722c2c2c4c2c2c2c313133622c2c353538662c2c2c2c2c302c302c2c2c3b
2015-11-27 10:28:13 DEBUG: [43ADFDC8: 5001 < 189.40.72.181] HEX: 696d65693a3335393731303034393037353039372c747261636b65722c2c2c4c2c2c2c313133622c2c353538662c2c2c2c2c302c302c2c2c3b
2015-11-27 10:28:38 DEBUG: [43ADFDC8: 5001 < 189.40.72.181] HEX: 3335393731303034393037353039373b
2015-11-27 10:28:38 DEBUG: [43ADFDC8: 5001 > 189.40.72.181] HEX: 4f4e
2015-11-27 10:29:13 DEBUG: [43ADFDC8: 5001 < 189.40.72.181] HEX: 3335393731303034393037353039373b
2015-11-27 10:29:13 DEBUG: [43ADFDC8: 5001 > 189.40.72.181] HEX: 4f4e
2015-11-27 10:30:13 DEBUG: [43ADFDC8: 5001 < 189.40.72.181] HEX: 3335393731303034393037353039373b
2015-11-27 10:30:13 DEBUG: [43ADFDC8: 5001 > 189.40.72.181] HEX: 4f4e
